src/models/Observation.js

```diff
@@ -157,40 +161,51 @@ class Observation {
     const user = realm.objectForPrimaryKey( "User", Number( id ) );
     obs.user = user;
 
-    const newLocalRecord = {
-      _created_at: new Date( ),
-      _synced_at: null,
+    const timestamps = {
       _updated_at: new Date( )
     };
+
+    const isSavedObservation = realm.objectForPrimaryKey( "Observation", obs.uuid );
+
+    if ( !isSavedObservation ) {
+      timestamps._created_at = new Date( );
+      timestamps._synced_at = null;
+    }
+
+    const addTimestampsToEvidence = ( evidence ) => {
+      // right now there isn't a way to edit photos or sounds via ObsEdit
+      // so we only need to add timestamps on the first time a local observation is saved
+      if ( !isSavedObservation ) {
+        evidence && evidence.map( record => {
+          return {
+            ...timestamps,
+            ...record
+          };
+        } );
+      }
+      return evidence;
+    };
+
     const taxon = obs.taxon ? Taxon.mapApiToRealm( obs.taxon ) : null;
-    const observationPhotos = obs.observationPhotos && obs.observationPhotos.map( photo => {
-      return {
-        ...newLocalRecord,
-        ...photo
-      };
-    } );
-    const observationSounds = obs.observationSounds && obs.observationSounds.map( sound => {
-      return {
-        ...newLocalRecord,
-        ...sound
-      };
-    } );
+    const observationPhotos = addTimestampsToEvidence( obs.observationPhotos );
+    const observationSounds = addTimestampsToEvidence( obs.observationSounds );
 
     const obsToSave = {
       ...obs,
-      ...newLocalRecord,
+      ...timestamps,
       taxon,
       observationPhotos,
-      observationSounds,
-      quality_grade: "needs_id"
+      observationSounds
     };
 
     realm?.write( ( ) => {
       // using 'modified' here for the case where a new observation has the same Taxon
       // as a previous observation; otherwise, realm will error out
+      // also using modified for updating observations which were already saved locally
       realm?.create( "Observation", obsToSave, "modified" );
     } );
-    return realm.objectForPrimaryKey( "Observation", obs.uuid );
+    const observation = realm.objectForPrimaryKey( "Observation", obs.uuid );
+    return observation;
```

Comment:
Just curious, was there a reason for doing this? Vestige of some logging, perhaps?